bindings/nodejs/CHANGELOG.md

```diff
@@ -19,6 +19,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Security -->
 
+## 1.1.3 - 2023-10-27
```

Comment:
So 1.1.2 has been released bu without prebuilds ?

Reply:
Yes https://www.npmjs.com/package/@iota/sdk/v/1.1.2

Reply:
Hm but maybe we could also just run without bumping the version, there was no release to gh, but not sure if the workflow fails if it was already published to npm